.github/workflows/test_examples.yml

```diff
@@ -122,6 +135,17 @@ jobs:
             examples/micro-speech/robot_output.xml
             examples/micro-speech/*.dump
 
+      - name: Close Invocation (failed)
+        if: ${{ failure() || cancelled() }}
+        run: $GITHUB_WORKSPACE/.github/scripts/manage_invocation.py close 1
+
+      - name: Close Invocation (success)
+        run: $GITHUB_WORKSPACE/.github/scripts/manage_invocation.py close 0
```

Comment:
how do we get logs into `Invocation log`?

Reply:
As per [distant-rs examples](https://github.com/antmicro/distant-rs/blob/master/examples/up_inv.py#L15), we can do it by sending the file named `build.log` using the `send_file(name, path)` method of the `Invocation` class.

However, we don't seem to have a single log file (we're running multiple robot files with each having its own "fake" target with a log). Unless we do want to upload something?

Reply:
I introduced a separate job which downloads the build log of the previous (actual job), uploads it to ResultStore and finalizes the invocation with the appropriate status. It is a bit hacky, but it works.